downloads/swan-lake-release-notes/swan-lake-2201.5.2/RELEASE_NOTE.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.5.2 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-5-2/
+active: 2201-5-2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-5-2
+    - /downloads/swan-lake-release-notes/2201.5.2/
+    - /downloads/swan-lake-release-notes/2201-5-2-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-5-2-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.5.2
+
+<em>Swan Lake 2201.5.2 is the second patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language and tooling.</em>
```

Comment:
```suggestion
<em>Swan Lake 2201.5.2 is the second patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language and tooling.</em>
```
Remove tooling if we don't have any tooling updates in this release.

Reply:
Updated